tests/output/ConsoleInteractionTest.py

```diff
@@ -613,6 +618,8 @@ def test_print_bears_no_settings(self):
             print_bears(self.log_printer.printer, bears, False)
             expected_string = "SomeBear:\n"
             expected_string += "  Some Description.\n\n"
+            expected_string += ("  The bear does not provide information "
```

Comment:
the whole bear information is indented as all the other paragraphs since it's a long list of bears :)